src/components/layout/MenuBar/MenuBar.js

```diff
@@ -5,47 +5,77 @@ import ProductSearch from '../../features/ProductSearch/ProductSearch';
 
 import styles from './MenuBar.module.scss';
 
-const MenuBar = ({ children }) => (
-  <div className={styles.root}>
-    <div className='container'>
-      <div className='row align-items-center'>
-        <div className='col'>
-          <ProductSearch />
-        </div>
-        <div className={'col-auto ' + styles.menu}>
-          <ul>
-            <li>
-              <a href='/#' className={styles.active}>
-                Home
-              </a>
-            </li>
-            <li>
-              <a href='/#'>Furniture</a>
-            </li>
-            <li>
-              <a href='/#'>Chair</a>
-            </li>
-            <li>
-              <a href='/#'>Table</a>
-            </li>
-            <li>
-              <a href='/#'>Sofa</a>
-            </li>
-            <li>
-              <a href='/#'>Bedroom</a>
-            </li>
-            <li>
-              <a href='/#'>Blog</a>
-            </li>
-          </ul>
+class MenuBar extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+  };
+
+  constructor(props) {
+    super(props);
+    this.toggleClass = this.toggleClass.bind(this);
+    this.state = {
+      active: false,
+    };
+  }
+
+  toggleClass() {
+    const currentState = this.state.active;
+    this.setState({ active: !currentState });
+  }
+
+  render() {
+    return (
+      <div className={styles.root}>
+        <div className={'container' + ' ' + styles.container}>
+          <div className={'row align-items-center' + ' ' + styles.divContainer}>
+            <div className={'col' + ' ' + styles.productSearch}>
+              <ProductSearch />
+              {/*className={this.state.active ? 'active' : null}*/}
```

Comment:
Hej! Usuń proszę ten komentarz

Reply:
ok, usunięte :) zapomniałem o nim podczas pracy, to był zbędny komentarz